docs/api/api_blueprint.apib

```diff
@@ -14111,7 +14111,7 @@ Related to ticket holders(attendees) of an event (free, paid, donation) to the e
 | `checkin-times` | Comma separated checkin times | string | - |
 | `pdf-url` | pdf url of the Attendee | url | - |
 
-## Attendees Collection [/v1/attendees/{event_identifier}{?page%5bsize%5d,page%5bnumber%5d,sort,filter}]
+## Attendees Collection [/v1/attendees/{?page%5bsize%5d,page%5bnumber%5d,sort,filter}]
```

Comment:
Please explain this change

Reply:
@shubham-padia The related event is provided in the JSON body itself. I think we missed when we corrected some endpoints.

Reply:
@shubham-padia The api doesn't take the event id from the url (as wrongly mentioned in the docs). So I removed it as per discussion with @iamareebjamal and @poush in the issue thread.